cadl-tests/package.json

```diff
@@ -14,7 +14,8 @@
     "@azure-tools/cadl-ranch-expect": "latest",
     "@azure-tools/cadl-ranch-specs": "latest",
     "@azure-tools/cadl-ranch-api": "latest",
-    "@azure-tools/cadl-java": "file:/../cadl-extension/azure-tools-cadl-java-0.1.0-dev.7.tgz"
+    "@azure-tools/cadl-java": "file:/../cadl-extension/azure-tools-cadl-java-0.1.0-dev.7.tgz",
+    "@cadl-lang/openapi": "latest"
```

Comment:
add this because cadl-ranch dev-driven test use it: https://github.com/Azure/cadl-ranch/blob/main/packages/cadl-ranch-specs/http/resiliency/dev-driven/main.cadl#L2